gamemodes/terrortown/entities/weapons/weapon_zm_carry.lua

```diff
@@ -58,88 +58,62 @@ SWEP.AllowDelete = false
 SWEP.AllowDrop = false
 SWEP.NoSights = true
 
-SWEP.EntHolding = nil
 SWEP.CarryHack = nil
 SWEP.Constr = nil
 SWEP.PrevOwner = nil
 
--- ConVar syncing
-if SERVER then
-
-	---
-	-- @realm server
-	local cvAllowRagCarry = CreateConVar("ttt_ragdoll_carrying", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @realm server
-	local cvPropForce = CreateConVar("ttt_prop_carrying_force", "60000", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @realm server
-	local cvPropThrow = CreateConVar("ttt_prop_throwing", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @realm server
-	local cvAllowRagPin = CreateConVar("ttt_ragdoll_pinning", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @realm server
-	local cvAllowRagPinInno = CreateConVar("ttt_ragdoll_pinning_innocents", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @note Allowing weapon pickups can allow players to cause a crash in the physics
-	-- system (ie. not fixable). Tuning the range seems to make this more
-	-- difficult. Not sure why. It's that kind of crash.
-	-- @realm server
-	local cvAllowWepCarry = CreateConVar("ttt_weapon_carrying", "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	---
-	-- @note Allowing weapon pickups can allow players to cause a crash in the physics
-	-- system (ie. not fixable). Tuning the range seems to make this more
-	-- difficult. Not sure why. It's that kind of crash.
-	-- @realm server
-	local cvWepCarryRange = CreateConVar("ttt_weapon_carrying_range", "50", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-
-	hook.Add("TTT2SyncGlobals", "TTT2SyncCarryGlobals", function()
-		SetGlobalBool(cvAllowRagCarry:GetName(), cvAllowRagCarry:GetBool())
-		SetGlobalInt(cvPropForce:GetName(), cvPropForce:GetInt())
-		SetGlobalBool(cvPropThrow:GetName(), cvPropThrow:GetBool())
-		SetGlobalBool(cvAllowRagPin:GetName(), cvAllowRagPin:GetBool())
-		SetGlobalBool(cvAllowRagPinInno:GetName(), cvAllowRagPinInno:GetBool())
-
-		SetGlobalBool(cvAllowWepCarry:GetName(), cvAllowWepCarry:GetBool())
-		SetGlobalInt(cvWepCarryRange:GetName(), cvWepCarryRange:GetInt())
-	end)
+--[[ THE GREAT UNMAGIC-ING ]]
```

Comment:
not our comment style. Besides that I don't get what you're trying to say with that